.eslintrc

```diff
@@ -83,6 +83,9 @@ rules:
   # Custom rules in tools/eslint-rules
   require-buffer: 2
 
+  # Suggest using `const` wherever possible
+  prefer-const: 2
+
```

Comment:
Oh, btw: We try to section rules like on this page: http://eslint.org/docs/rules/. Could you move it above the custom rules and add modify it like this:

```
# ECMAScript 6
# list: http://eslint.org/docs/rules/#ecmascript-6
## Suggest using 'const' wherever possible
prefer-const: 2
```